x-pack/plugins/alerting/server/usage/lib/get_telemetry_from_kibana.ts

```diff
@@ -0,0 +1,317 @@
+                      // using of regex is not allowed in painless language
+                      parsed = Integer.parseInt(throttle);
+  
+                      if (timeChar.equals("s")) {
+                        parsed = parsed;
+                      } else if (timeChar.equals("m")) {
+                        parsed = parsed * 60;
+                      } else if (timeChar.equals("h")) {
+                        parsed = parsed * 60 * 60;
+                      } else if (timeChar.equals("d")) {
+                        parsed = parsed * 24 * 60 * 60;
+                      }
+                      emit(parsed);
+                    }
+                }
+              }
+              emit(parsed);
+              `,
+            },
+          },
+        },
+        aggs: {
+          by_rule_type_id: {
+            terms: {
+              field: 'alert.alertTypeId',
+              size: NUM_ALERTING_RULE_TYPES,
+            },
+          },
+          max_throttle_time: { max: { field: 'rule_throttle_interval' } },
+          min_throttle_time: { min: { field: 'rule_throttle_interval' } },
+          avg_throttle_time: { avg: { field: 'rule_throttle_interval' } },
+          max_interval_time: { max: { field: 'rule_schedule_interval' } },
+          min_interval_time: { min: { field: 'rule_schedule_interval' } },
+          avg_interval_time: { avg: { field: 'rule_schedule_interval' } },
+          max_actions_count: { max: { field: 'rule_action_count' } },
+          min_actions_count: { min: { field: 'rule_action_count' } },
+          avg_actions_count: { avg: { field: 'rule_action_count' } },
+        },
+      },
+    };
+
+    logger.debug(`query for getTotalCountAggregations - ${JSON.stringify(query)}`);
+    const results = await esClient.search(query);
+
+    logger.debug(`results for getTotalCountAggregations query - ${JSON.stringify(results)}`);
+
+    const aggregations = results.aggregations as {
+      by_rule_type_id: AggregationsTermsAggregateBase<AggregationsStringTermsBucketKeys>;
+      max_throttle_time: AggregationsSingleMetricAggregateBase;
+      min_throttle_time: AggregationsSingleMetricAggregateBase;
+      avg_throttle_time: AggregationsSingleMetricAggregateBase;
+      max_interval_time: AggregationsSingleMetricAggregateBase;
+      min_interval_time: AggregationsSingleMetricAggregateBase;
+      avg_interval_time: AggregationsSingleMetricAggregateBase;
+      max_actions_count: AggregationsSingleMetricAggregateBase;
+      min_actions_count: AggregationsSingleMetricAggregateBase;
+      avg_actions_count: AggregationsSingleMetricAggregateBase;
+    };
+
+    const totalRulesCount =
+      typeof results.hits.total === 'number' ? results.hits.total : results.hits.total?.value;
+
+    return {
+      count_total: totalRulesCount ?? 0,
+      count_by_type: parseSimpleRuleTypeBucket(aggregations.by_rule_type_id.buckets),
+      throttle_time: {
+        min: `${aggregations.min_throttle_time.value}s`,
```

Comment:
When I grab the telemetry before any rules have run, `aggregations.min_throttle_time.value` etc are `null`, so the telemetry shows:

```
  "throttle_time": {
    "min": "nulls",
    "avg": "nulls",
    "max": "nulls"
  },
  "schedule_time": {
    "min": "nulls",
    "avg": "nulls",
    "max": "nulls"
  },
  "throttle_time_number_s": {
    "min": 0,
    "avg": 0,
    "max": 0
  },
  "schedule_time_number_s": {
    "min": 0,
    "avg": 0,
    "max": 0
  },
```